types/braintree-web/test/web.ts

```diff
@@ -640,6 +640,18 @@ braintree.client.create(
                     })
                     .then(paypalCheckout => {
                         // window.paypal.Buttons is now available to use
+                        window.paypal.Buttons({
+                            onApprove(data) {
+                                return paypalCheckout.tokenizePayment(data);
+                            },
+                            onInit(data) {
+                                paypalCheckout.updatePayment({
+                                    amount: "10.00",
+                                    currency: "USD",
+                                    paymentId: data.paymentId,
+                                });
```

Comment:
It would be good to check promise gets returned here:

```suggestion
                                }).then(res => res);
```

Reply:
Thanks, I've made a commit to add this.